### DISCUSSION TITLE
Welcome to custom-web-browser-and-server Discussions!

### DISCUSSION THREAD

**Discussion by marceldobehere** [marceldobehere/custom-web-browser-and-server#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RetroDude128** [comment]
Guys I'm legit gonna troll in #general

**Comment by RetroDude128** [comment]
Yo dude you should totally give me the world seed to your minecraft server
